dockerfiles/run/guice/cassandra-rabbitmq/destination/conf/blobstore.properties

```diff
@@ -1,3 +1,5 @@
+objectstore.implementation=swift
```

Comment:
and what's the point of prefixing by objectstore ? `implementation`  or `backend` should be enough